### ISSUE TITLE
Support inference of entry.request as POST in transaction for bundle resource

### ISSUE THREAD

**Issue by citizenrich** [jamesagnew/hapi-fhir#991]
tl;dr: I receive an error in trying to create resources using type transaction for a bundle. This is tested on the hosted HAPI server. Error:

"Transaction bundle entry has missing or invalid HTTP Verb specified in Bundle.entry(0).request.method. Found value: \"null\""

To reproduce:
* Generate one bundle of synthetic records using Synthea (latest build not the hosted API).
* Change the type of the bundle to 'transaction'.
* cURL:
curl -X POST -H "Content-Type: application/json" --data @data.json "http://hapi.fhir.org/baseDstu3/"

I do not add entry.request before POSTing. According to the spec (though it's a bit confusing perhaps) the server SHALL infer the request type itself as long as there is a resource for POST.
https://www.hl7.org/fhir/bundle.html#transaction

Could very well be doing something wrong.

**Comment by patrick-werner** [comment]
you are right, this is not supported in HAPI right now. As we (MOLIT Institut) also want this functionality i will start to have a look at this feature after i finished the $document operation.

- [ ] infering the request
- [ ] allow POST of document bundles onto the server base URL

**Comment by citizenrich** [comment]
Is this an authoritative ‘won’t fix’ or a simply the bot doing its thing, or already implemented?

**Comment by jamesagnew** [comment]
This is just the bot closing after 2 years of inactivity. This is pretty
lowe priority since there is an obvious workaround but I'm not opposed to
fixing it. Feel free to reopen.

**Comment by Ayinks** [comment]
Hello, please is this issue now fixed? I am having the same issue POSTing both transaction and document-type bundles to server base URL. You mentioned there is an obvious workaround, could you please give more details on the likeworkaround?

Thank you

**Comment by s7jung** [comment]
> Hello, please is this issue now fixed? I am having the same issue POSTing both transaction and document-type bundles to server base URL. You mentioned there is an obvious workaround, could you please give more details on the likely workaround?
> 
> Thank you

have you figured this out?
